data/json/items/ranged/crossbows.json

```diff
@@ -355,21 +355,20 @@
   },
   {
     "id": "crossbow",
-    "//": "Tileset whitelist for crossbows",
+    "//": "150lb @ 10in powerstroke, 34in wood limbs, 37.7J, 0.337 slugs with a 0.066lb bolt",
```

Comment:
is this intentional? and the other one as well?

Reply:
Looks like the script was losing one of the comments for objects that had the `//` key twice.

Reply:
FWIW, we shouldn't have duplicate keys like this (for this reason).  They should probably be migrated to use `"//"`, `"//2"`, etc.  You don't have to do that on this PR, but it might be helpful since you've found some anyway.